lib/api/client-commands/network/captureNetworkRequests.js

```diff
@@ -19,15 +19,19 @@ const {Logger} = require('../../utils');
  *    });
  *  });
  *
- * @method captureNetworkRequests
- * @syntax .captureNetworkRequests(onRequestCallback)
+ * @method network.captureNetworkRequests
+ * @syntax .network.captureNetworkRequests(onRequestCallback)
```

Comment:
Should be `network.captureRequests`. And for `@syntax`, we can include both the syntaxes:
```js
@syntax .captureNetworkRequests(onRequestCallback)
@syntax .network.captureRequests(onRequestCallback)
```

Same for the other files as well.